ERCS/erc-7858.md

```diff
@@ -0,0 +1,221 @@
+---
+eip: 7858
+title: Expirable NFTs and SBTs
+description: Non-fungible (NFT) and soulbound (SBT) tokens with expiration, supporting time-limited use cases.
+author: sirawt (@MASDXI), ADISAKBOONMARK (@ADISAKBOONMARK), parametprame (@parametprame), Nacharoen (@najaroen)
+discussions-to: https://ethereum-magicians.org/t/erc-7858-expirable-nft-sbt/22406
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-01-04
+requires: 165, 721
+---
+
+## Abstract
+
+Introduces an extension for [ERC-721](./eip-721.md) Non-Fungible Tokens (NFTs) and Soulbound Tokens (SBTs) that adds an expiration mechanism, allowing tokens to become invalid after a predefined period. This additional layer of functionality ensures that the expiration mechanism does not interfere with existing NFTs or SBTs, preserving transferability for NFTs and compatibility with current DApps such as NFT Marketplace. Expiration can be defined using either block height or timestamp, offering flexibility for various use cases.
+
+## Motivation
+
+Introduces an extension for [ERC-721](./eip-721.md) Non-Fungible Tokens (NFTs) and Soulbound Tokens (SBTs), which facilitates the implementation of an expiration mechanism.
```

Comment:
You're just repeating your abstract here. It's safe to assume that someone reading your motivation has read your abstract.